docs/advanced/10_git_sync/index.mdx

```diff
@@ -0,0 +1,48 @@
+import DocCard from '@site/src/components/DocCard';
+
+# Sync Workspace to Remote Git Repository
+
+From the workspace settings, you can set a [git_repository](../../integrations/git_repository.mdx) resource on which the workspace will automatically commit and push scripts, flows and apps to the repository on each deploy.
+
+<iframe
+	style={{ aspectRatio: '16/9' }}
+	src="https://www.youtube.com/embed/JqG0KNYWLx0?vq=hd1080"
+	title="Sync Workspace to Remote Git Repository"
+	frameBorder="0"
+	allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+	allowFullScreen
+	className="border-2 rounded-xl object-cover w-full dark:border-gray-800"
+></iframe>
+
+<br/>
+
+:::info Private Folders
+
+Script, flows and apps in the user private folders will be ignored 
```

Comment:
nit: we can prob remove "Script, flows and apps in the user private folders will be ignored", it's repetitive with the below